debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleBlobDataTypesIT.java

```diff
@@ -950,6 +952,66 @@ public void shouldNotStreamAnyChangesWhenLobEraseIsDetected() throws Exception {
         assertNoRecordsToConsume();
     }
 
+    @Test
+    @FixFor({ "DBZ-2948", "DBZ-5773" })
+    @SkipWhenAdapterNameIsNot(value = SkipWhenAdapterNameIsNot.AdapterName.OLR, reason = "OpenLogReplicator does not differentiate between LOB operations")
+    public void shouldStreamChangesWhenLobEraseIsDetected() throws Exception {
+        String ddl = "CREATE TABLE BLOB_TEST ("
```

Comment:
Is this suppurted for XStream?

Reply:
There is now two methods:
* shouldNotStreamAnyChangesWhenLobEraseIsDetected
* shouldStreamChangesWhenLobEraseIsDetected

The former works for LogMiner/XStream and the later only works for OpenLogReplicator.  The new test added was to show that when using `DBMS_LOB.ERASE` that we cannot differentiate between the LOB operations.